recipes-bsp/txe-secure-boot/smmstoretool_git.bb

```diff
@@ -0,0 +1,31 @@
+SUMMARY = "Offline SMMSTORE variable modification tool"
```

Comment:
Please move it under coreboot-utils.
Then, the txe-secure-boot directory can be removed.